File: SeleniumExercise/src/main/java/api/dtos/LoginDTO.java
package api.dtos;

public class LoginDTO {
	public String apiUser;
	public String apiKey;
	public Long userId;
	public LoginType[] loginTypes;
	public Long sessionId;
	public String sessionKey;
	public String analyticsEndpoint;
	public String location;
	public Boolean websocketsEnabled;
	public Boolean twoAuthEnabled;
	public Boolean expired;
	public String token;
	public String authError;
	public String subdomain;
	public CustomerPropertyDTO[] uiProperties;
	public UserPropertyDTO[] userProperties;
	public Boolean googleAnalyticsEnabled;
	public String venaSupportUser;
	public String salesforceId;
	public String tenantType;
	public Long hubspotContactId;
	public String trialVersion;
	public String status;
	public String phoneNumber;
	public boolean canChangePassword;

	public enum LoginType {
		ANY, CONTRIBUTOR, MANAGER, ADMIN, MODELER, DASHBOARDER, REPORTER
	}
}

File: SeleniumExerciseC#/api/dtos/LoginDTO.cs
public record LoginDTO
{
    public string apiUser { get; init; }
    public string apiKey { get; init; }
    public string userId { get; init; }
    public string sessionId { get; init; }
    public string sessionKey { get; init; }
    public string analyticsEndpoint { get; init; }
    public string location { get; init; }
    public Boolean websocketsEnabled { get; init; }
    public Boolean twoAuthEnabled { get; init; }
    public Boolean expired { get; init; }
    public string token { get; init; }
    public string authError { get; init; }
    public string subdomain { get; init; }
    public CustomerPropertyDTO[] uiProperties { get; init; }
    public UserPropertyDTO[] userProperties { get; init; }
    public Boolean googleAnalyticsEnabled { get; init; }
    public string venaSupportUser { get; init; }
    public string salesforceId { get; init; }
    public string tenantType { get; init; }
    public string hubspotContactId { get; init; }
    public string trialVersion { get; init; }
    public string status { get; init; }
    public string phoneNumber { get; init; }
    public bool canChangePassword { get; init; }
    public string organizationType { get; init; }
}

public enum LoginType
{
    ANY, CONTRIBUTOR, MANAGER, ADMIN, MODELER, DASHBOARDER, REPORTER
}